rlp/decode_test.go

```diff
@@ -806,9 +806,8 @@ func ExampleDecode() {
 	input, _ := hex.DecodeString("C90A1486666F6F626172")
 
 	type example struct {
-		A, B    uint
-		private uint // private fields are ignored
```

Comment:
Isn't this also a testing field, that it gets ignored?

Reply:
I think it's better to remove this field instead of adding an
ugly nolint comment because this code is an example and the code
is visible in godoc.

Reply:
Your decision.